pkg/catalog/helpers_test.go

```diff
@@ -9,7 +9,6 @@ import (
 	specs "github.com/servicemeshinterface/smi-sdk-go/pkg/apis/specs/v1alpha4"
 	split "github.com/servicemeshinterface/smi-sdk-go/pkg/apis/split/v1alpha2"
 	corev1 "k8s.io/api/core/v1"
```

Comment:
This is a good example of why aliasing is helpful, because multiple imports packages could be named `v1` and changing the import alias later on is a bit annoying.